fire/cli/info.py

```diff
@@ -382,25 +384,52 @@ def punkt(ident: str, obs: str, koord: str, detaljeret: bool, **kwargs) -> None:
 
 
 @info.command()
+@click.option(
+    "--alle", is_flag=True, default=False, help="Udskriv også tidsserier",
+)
 @fire.cli.default_options()
-@click.argument("srid")
-def srid(srid: str, **kwargs):
+@click.argument("srid", required=False)
+def srid(srid: str, alle: bool, **kwargs):
     """
     Information om et givent SRID (Spatial Reference ID)
 
     Eksempler på SRID'er: EPSG:25832, DK:SYS34, TS:81013
+
+    Anføres ikke SRID gives liste af mulige SRID. Som standard uden lokale
+    tidsseriekoordinatsystemer
+
+    tilvalg ``--alle` kan kun vælges uden angiven SRID. Udvider listen med 
+    lokale tidsseriekoordinatsystemer
     """
-    srid_name = srid
+    if not srid:
+        srid_db = firedb.session.query(Srid).all()
 
-    try:
-        srid = firedb.hent_srid(srid_name)
-    except NoResultFound:
-        fire.cli.print(f"Error! {srid_name} not found!", fg="red", err=True)
-        sys.exit(1)
+        srid_list = []
+
+        for srid_item in srid_db:
+            condition = False
+            if not alle:
+                condition = re.search(r"^TS:", srid_item.name)
+
+            if not condition:
+                srid_list.append("{:20}".format(srid_item.name) + srid_item.beskrivelse)
+
+        srid_list.sort()
+        for srid_line in srid_list:
+            fire.cli.print(srid_line)
```

Comment:
Med ovenstående ændring kan du forsimple de her linjer:
```suggestion
        for srid_item in srid_db:
            fire.cli.print(f"{srid_item.name:20}" + srid_item.beskrivelse)
```

Fidusen er, at du lader databasen gøre det hårde arbejde. Det betyder ikke alverden i det her tilfælde, da der er tale om små datamængder men det er en god vane at indarbejde.